### PR TITLE
[O2B-706] Add fields ccdb by run number

### PR DESCRIPTION
**Pull request by JikHuijberts**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected
- [x] `CHANGELOG.md` and `database/CHANGELOG.md` files were updated

### CODE REVIEW

File: lib/database/models/typedefs/SequelizeRun.js [hunk 1]
```diff
@@ -53,9 +53,18 @@
...
+ * @property {string|null} startOfDataTransfer
+ * @property {string|null} endOfDataTransfer
+ * @property {string|null} ctfFileCount
+ * @property {string|null} ctfFileSize
+ * @property {string|null} tfFileCount
+ * @property {string|null} tfFileCount
+ * @property {string|null} tfFileSize
+ * @property {string|null} otherFileCount
+ * @property {string|null} otherFileSize
+ * @property {number|null} runDuration
```
Comment (martinboulais) [inline lib/database/models/typedefs/SequelizeRun.js:65]: Are you sure those are string? And not Bigint or anything?

File: lib/database/models/typedefs/SequelizeRun.js [hunk 2]
```diff
@@ -53,9 +53,18 @@
...
  * @property {string} createdAt
  * @property {string} updatedAt
- * @property {number|null} runDuration
```
Comment (martinboulais) [inline lib/database/models/typedefs/SequelizeRun.js:58]: Please leave this here, I ordered fields to have them in the database order then the generated fields

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,7 +19,7 @@ All notable changes to this project will be documented in this file. See [standa
   * Pagination limits is now of 1000 items in GUI and 5k in the API and export
 * Notable changes for developers:
   * Some entities and sequelize models have typedefs to help autocompletion, code navigation and refactoring
-
+
```
Comment (martinboulais) [inline CHANGELOG.md:22]: I think this tab must be removed

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (JikHuijberts) [reply CHANGELOG.md]: Removed!

File: lib/database/models/typedefs/SequelizeRun.js (hunk 2 above)
Comment (JikHuijberts) [reply lib/database/models/typedefs/SequelizeRun.js:58]: Done

File: lib/database/models/typedefs/SequelizeRun.js (hunk 1 above)
Comment (JikHuijberts) [reply lib/database/models/typedefs/SequelizeRun.js]: They get returned as number | string | null depending on if the number type can handle it or if it's empty or not. So I updated it to be according to that info

**Review by martinboulais — DISMISSED** [review]

**Review by martinboulais — APPROVED** [review]

**Pull request merged**